docs/NNNN-n-tuple-omit-prefix-storage-layout.md

```diff
@@ -0,0 +1,183 @@
+The object id is then divided into N n-tuple segments, and used to create nested paths under the OCFL storage root, followed by the prefix-omitted object id directory.  
+
+This layout combines the advantages of 0006-flat-omit-prefix-storage-layout (directory name transparency) and the 0004-hashed-n-tuple-storage-layout (enhanced file system/bucket performance). 
+
+The limitations of this layout are filesystem dependent, but are generally as follows:
+
+* The size of object identifiers, minus the length of the prefix, cannot exceed the maximum allowed directory name size
+  (eg. 255 characters)
+* Object identifiers cannot include characters that are illegal in directory names
+
+## Parameters
+
+### Summary
+
+* **Name:** `delimiter`
+  * **Description:** The case-insensitive, delimiter marking the end of the OCFL object identifier prefix; MUST consist
+    of a character string of length one or greater. If the delimiter is found multiple times in the OCFL object
+    identifier, its last occurence (right-most) will be used to select the termination of the prefix.
+  * **Type:** string
+  * **Constraints:** Must not be empty
+  * **Default:** :
+* **Name**: `tupleSize`
+  * **Description:** Indicates the segment size (in characters) to split the
+    identifier into
+  * **Type:** number
+  * **Constraints:** An integer between 1 and 32 inclusive
+  * **Default:** 3
+* **Name:** `numberOfTuples`
+  * **Description:** Indicates the number of segments to use for path generation
+  * **Type:** number
+  * **Constraints:** An integer between 1 and 32 inclusive
+  * **Default:** 3
+* **Name:** `zeroPadding`
+  * **Description:** Indicates whether to use left or right zero padding for ids less than tupleSize * numberOfTuples
+  * **Type:** string
+  * **Constraints:** Must be either "left", "right" or "none"
+  * **Default:** left
+* **Name:** `reverseObjectRoot`
+  * **Description:** When true, indicates that the prefix-omitted, padded object identifier should be reversed
+  * **Type:** boolean
+  * **Default:** false
+
+## Procedure
+
+The following is an outline of the steps to map an OCFL object identifier to an
+OCFL object root path:
+
+1. Remove the prefix, which is everything to the left of the right-most instance of the delimiter, as well as the delimiter. If there is no delimiter, the whole id is used; if the delimiter is found at the end, an error is thrown.
+2. Optionally, add zero-padding to the left or right of the remaining id, depending on `zeroPadding` configuration.
+3. Optionally reverse the remaining id, depending on `reverseObjectRoot`
+4. Starting at the leftmost character of the resulting id and working right, divide the id into `numberOfTuples` each containing `tupleSize` characters.
+5. It is expected that the UTF-8 identifier from step 5 is split between grapheme clusters
+6. Join the tuples, in order, using the filesystem path separator.
```

Comment:
Create the start of the object root path by joining the triples...

Reply:
I made this change as requested